thing-url-adapter.js

```diff
@@ -264,8 +283,20 @@ class ThingURLDevice extends Device {
     if (this.closing) {
       return;
     }
+    // if we have an entry for the wss url stub in jwt_auth
+    // add auth to query string
+    let auth = '';
+    for (const i in JWT_AUTH) {
+      if (this.wsUrl.includes(i)) {
+        auth = `?Authorization=Bearer ${JWT_AUTH[i]}`;
+        break;
+      }
+    }
+    // -- added '' and {} to get eslint to shut up
+    if (this.ws === null) {
```

Comment:
Why is the null check now necessary here?

Reply:
If there is no jwt configured for the thing we are trying to connect to it ws will be null when we get here. I'll add a comment to the file.